contracts/erc20/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@0x/contracts-erc20",
-    "version": "3.3.14",
+    "version": "3.3.15",
```

Comment:
no need to manually update this, instead you should add an entry to `contracts/erc20/CHANGELOG.json` like:
```
    {
        "version": "3.3.15",
        "changes": [
            {
                "note": "Add ethers to dependencies",
                "pr": 305
            }
        ]
    },
```
(note that you don't need to put the `timestamp`)
The GitHub `publish` action will handle the rest 🙂

Reply:
yall are fancy. updated.